spec/spec_helper.rb

```diff
@@ -0,0 +1,109 @@
+require 'omniauth'
+
+OmniAuth.config.test_mode = true
+
+omniauth_hash = OmniAuth::AuthHash.new({
+  'provider': 'auth_provider',
+  'uid':      '00001',
+  'info': {
+    'name':       'Hello World',
+    'email':      'hello_world@example.com',
+    'first_name': 'Hello',
+    'last_name':  'World'
+  }
+})
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.